framework/decode/openxr_replay_consumer_base.cpp

```diff
@@ -70,10 +70,70 @@ void OpenXrReplayConsumerBase::SetVulkanReplayConsumer(VulkanReplayConsumerBase*
     vulkan_replay_consumer_ = vulkan_replay_consumer;
 }
 
+static XRAPI_ATTR XrResult XRAPI_CALL ReplayXrTermSetDebugUtilsObjectNameEXT(XrInstance,
+                                                                             const XrDebugUtilsObjectNameInfoEXT*)
+{
+    // This is a stub to prevent crashes during replay, see AddInstanceTable below.
+    return XR_SUCCESS;
+}
+
+static XRAPI_ATTR XrResult XRAPI_CALL ReplayXrTermCreateDebugUtilsMessengerEXT(
+    XrInstance, const XrDebugUtilsMessengerCreateInfoEXT*, XrDebugUtilsMessengerEXT*)
+{
+    // This is a stub to prevent crashes during replay, see AddInstanceTable below.
+    return XR_SUCCESS;
+}
+
+static XRAPI_ATTR XrResult XRAPI_CALL ReplayXrTermDestroyDebugUtilsMessengerEXT(XrDebugUtilsMessengerEXT)
+{
+    // This is a stub to prevent crashes during replay, see AddInstanceTable below.
+    return XR_SUCCESS;
+}
+
+static XRAPI_ATTR XrResult XRAPI_CALL
+ReplayXrTermSubmitDebugUtilsMessageEXT(XrInstance                                  instance,
+                                       XrDebugUtilsMessageSeverityFlagsEXT         messageSeverity,
+                                       XrDebugUtilsMessageTypeFlagsEXT             messageTypes,
+                                       const XrDebugUtilsMessengerCallbackDataEXT* callbackData)
+{
+    // This is a stub to prevent crashes during replay, see AddInstanceTable below.
+    return XR_SUCCESS;
+}
+
```

Comment:
Don't we also need stubs for the remaining entrypoints in that extension?  Otherwise they may get triggered and crash:
 * xrSessionBeginDebugUtilsLabelRegionEXT
 * xrSessionEndDebugUtilsLabelRegionEXT
 * xrSessionInsertDebugUtilsLabelEXT

Reply:
For some reason, the OpenXR loader doesn't terminate them, I mirrored its behavior.  I can add them if preferred.

Reply:
I think we should, just in case the loader gets updated since they've shown a desire to do it for the other entrypoints of the same extension.  Just to be safe.